oap-server/server-starter/src/main/resources/ui-initialized-templates/postgresql/postgresql-service.json

```
@@ -0,0 +1,700 @@
+                  "showUnit":true
+               },
+               "widget":{
+                  "title":"Max WAL Size (GB)"
+               },
+               "metricConfig":[
+                  {
+                     "calculation":"byteToGB"
+                  }
+               ]
+            },
+            {
+               "x":9,
+               "y":6,
+               "w":5,
+               "h":6,
+               "i":"7",
+               "type":"Widget",
+               "metricTypes":[
+                  "readMetricsValue"
+               ],
+               "metrics":[
+                  "meter_pg_max_parallel_workers"
+               ],
+               "graph":{
+                  "type":"Card",
+                  "fontSize":40,
+                  "textAlign":"center",
+                  "showUnit":true
+               },
+               "widget":{
+                  "title":"Max Parallel Workers"
+               }
+            },
+            {
+               "x":19,
+               "y":0,
+               "w":5,
+               "h":6,
+               "i":"8",
+               "type":"Widget",
+               "metricTypes":[
+                  "readMetricsValue"
+               ],
+               "metrics":[
+                  "meter_pg_max_worker_processes"
+               ],
+               "graph":{
+                  "type":"Card",
+                  "fontSize":40,
+                  "textAlign":"center",
+                  "showUnit":true
+               },
+               "widget":{
+                  "title":"Max Worker Processes"
+               }
+            },
+            {
+               "x":0,
+               "y":12,
+               "w":6,
+               "h":10,
+               "i":"9",
+               "type":"Widget",
+               "metricTypes":[
+                  "readLabeledMetricsValues"
+               ],
+               "metrics":[
+                  "meter_pg_fetched_rows_rate"
+               ],
+               "graph":{
+                  "type":"Area",
+                  "opacity":0.4,
+                  "showXAxis":true,
+                  "showYAxis":true
+               },
+               "widget":{
+                  "title":"Fetched Rows Trend (rows per minute)"
+               },
+               "value":"9",
+               "label":"9"
+            },
+            {
+               "x":6,
+               "y":12,
+               "w":6,
+               "h":10,
+               "i":"10",
+               "type":"Widget",
+               "metricTypes":[
+                  "readLabeledMetricsValues"
+               ],
+               "metrics":[
+                  "meter_pg_inserted_rows_rate"
+               ],
+               "graph":{
+                  "type":"Area",
+                  "opacity":0.4,
+                  "showXAxis":true,
+                  "showYAxis":true
+               },
+               "widget":{
+                  "title":"Inserted Rows Trend (rows per minute)"
```

```suggestion
                  "title":"Inserted Rows Trend (rows per second)"
```
I found your expression is:
```yaml
  - name: inserted_rows_rate
    exp: pg_stat_database_tup_inserted.sum(['datname','server']).rate('PT1M')
```
`rate(Duration)` means: Calculates the per-second `average rate of increase` in the time range, it should be `rows per second`
If you want to calculate `rows per minute`, you should use:
`increase(Duration)`: Calculates the increase in the time range.

I think using `rate` here is correct, just need to fix the unit, and please check your previous `MySQL monitoring PR` and make these consistently.

---

It's my fault, I'm going to calculate `per second`, I'll change it now.